scripts/main.js

```diff
@@ -0,0 +1,166 @@
+let main = null;
+let header = null;
+let customers = null;
+let navigation = null;
+let mobileMenuToggle = null;
+let featureSpeakersSection = null;
+// list os speakers inside array of objects
+const speakers = [
```

Comment:
Awesome ✔ The list of speakers used to generate the HTML is assigned to a variable as an array or object. The same list also contains an speaker object for each that holds their specific data.

Reply:
@mateo951 Thank you.